projects/swimlane/ngx-cron/src/lib/ngx-cron.service.ts

```diff
@@ -1,5 +1,6 @@
 import { Injectable } from '@angular/core';
-import { default as ExpressionDescriptor } from 'cronstrue';
+import { default as ExpressionDescriptor } from 'cronstrue/i18n';
```

Comment:
nitpick: why not just `import ExpressionDescriptor from 'cronstrue/i18n'`?

Reply:
Agree. Done